### PR TITLE
added more plugins

### PR DESCRIPTION
**Pull request by Peeeekay**

### CODE REVIEW

File: main.star [hunk 1]
```diff
@@ -161,7 +161,7 @@ def download_plugins(plan, plugins_dir, plugins_to_download, plugin_branch_to_us
     for plugin in plugins_to_download:
         url = plugins.get_plugin_url(plugin, plugin_branch_to_use, plugin_repo_to_use)
         plugin_filename = plugins.get_filename(plugin)
-        download_and_run_command = "mkdir /app/plugins && wget -O ./{0}/{1} {2}".format(plugins_dir, plugin_filename, url)
```
Comment (Peeeekay) [inline main.star:164]: wasn't able to include multiple plugins so fixed that. @h4ck3rk3y @laurentluce

Comment (Peeeekay) [reply main.star:164]: something like this was failing before:
kurtosis run . '{"OPENAI_API_KEY": "YOUR_API_KEY_HERE", "ALLOWLISTED_PLUGINS": "AutoGPT_YouTube,AutoGPTPMPlugin,AutoGPTWolframAlpha,AutoGPTTwitter"}'

Comment (h4ck3rk3y) [reply main.star:164]: looks good PK! Remember to raise PRs to these repos too :)

**Review by h4ck3rk3y — APPROVED** [review]

File: main.star (hunk 1 above)
Comment (Peeeekay) [reply main.star:164]: yepp; doing it after merging.

**Pull request merged**